.github/workflows/libcxx-check-generated-files.yml

```diff
@@ -0,0 +1,25 @@
+name: "Check libc++ generated files"
+on:
+  pull_request:
+    paths:
+      - 'libcxx/**'
+
+jobs:
+  check_generated_files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch LLVM sources
+        uses: actions/checkout@v4
+        with:
```

Comment:
You can remove this unless you need to use `git-clang-format`, it's only there so we can diff.

Reply:
The job does need to pin down the version of clang-format we're using. Are you saying that this is really needed only for `git-clang-format` and not for `clang-format` itself? If so, where do we get `clang-format 17` from? Does it just happen to be the one supplied with the base Ubuntu image?

Reply:
No I meant specifically `depth: 2` to checkout. It's not needed unless you need to format a diff.

Reply:
Oh, I see! Thanks, yeah I will remove that.